metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1498,6 +1543,30 @@ private void cancelNextWriteNoOpRecord() {
         queue.cancelDeferred(WRITE_NO_OP_RECORD);
     }
 
+    private static final String WRITE_REMOVE_DELEGATIONTOKEN_RECORD = "writeRemoveDelegationTokenRecord";
```

Comment:
I think should be something like "maybeExpireDelegationTokens" or something to reflect what this event is doing (rather than what record ends up being used)

Reply:
I changed it to SWEEP_EXPIRED_DELEGATION_TOKENS and "sweepExpiredDelegationTokens" as that is more of what it is doing.